docs/antora/ccutil.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -ex
+rm -rf build/ccutil
+mkdir -p build/ccutil
+
+cp -r ccutil/guides-server build/ccutil
+mkdir -p build/ccutil/guides-server
+cp -r build/assembler/guides-server build/ccutil/guides-server
+
+cp -r ccutil/guides-operator build/ccutil
+mkdir -p build/ccutil/guides-operator
+cp -r build/assembler/guides-operator build/ccutil/guides-operator
+# TODO: having the images here breaks the ccutil script with a Python error
+rm -rf build/ccutil/guides-operator/latest/_images
+sed -E -i '/^:imagesdir:/d' build/ccutil/guides-operator/guides-operator/20.0/guides-keycloak-operator-ccutil.adoc
+
+# sed -i 's|xref:{page-version}@guides-server::\([^.]*\)|https://ahus1.github.io/keycloak-antora/guides-server/latest/\1|g' guides-operator/guides-operator/20.0/guides-keycloak-operator.adoc
+
+IMAGENAME='quay.io/ivanhorvath/ccutil:amazing'
+
+if which podman &>/dev/null;
+then
+    CRT=$(which podman)
+else
+    CRT=$(which docker)
+fi
+
+if ! ${CRT} exec ccutil true &>/dev/null;
+then
+    ${CRT} rm -f ccutil
+    ${CRT} run --privileged --ulimit host -d -v $(pwd):/docs:rw --name ccutil ${IMAGENAME}
```

Comment:
This requires me to run Podman with `sudo`, which results in generated files not being possible to delete with my regular user afterwards. Changing to the following makes it run:

```suggestion
    ${CRT} run -d -v $(pwd):/docs:Z --name ccutil ${IMAGENAME}
```

I'm getting some errors though, but not sure if that's caused by this or something else

Reply:
For me, this runs without `sudo`, and this also avoids the issue you describe with the permissions. I'm running Fedora 36.

Can you share more about your environment and the error message you see?